src/app/lib/layoutGrid.js

```diff
@@ -0,0 +1,33 @@
+import { css } from 'styled-components';
+
+export const layoutGridWrapper = css`
+  display: grid;
+  @media (max-width: 1007px) {
+    grid-template-columns: repeat(6, 1fr);
+  }
+  @media (min-width: 1008px) and (max-width: 1279px) {
+    grid-template-columns: repeat(8, 1fr);
+  }
+  @media (min-width: 1280px) {
+    grid-template-columns: repeat(10, 1fr);
+  }
+`;
+
+export const layoutGridMain = css`
```

Comment:
I'd suggest renaming this to `layoutGridItem` and the `layoutGridFull` below to `layoutGridItemFullWidth`